src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -89,8 +96,31 @@ public PersonBuilder withEmail(String email) {
         return this;
     }
 
-    public Person build() {
-        return new Person(name, phone, email, address, tags);
+    /**
+     * Sets the {@code scheduleName} of the {@code Person} that we are building.
+     */
+    public PersonBuilder withScheduleName(String scheduleName) {
+        this.schedule = new Schedule(scheduleName, this.schedule.dateString, this.schedule.timeString);
+        return this;
+    }
+
+    /**
+     * Sets the {@code scheduleDate} of the {@code Person} that we are building.
+     */
+    public PersonBuilder withScheduleDate(String scheduleDate) {
+        this.schedule = new Schedule(this.schedule.scheduleName, scheduleDate, this.schedule.timeString);
+        return this;
+    }
+
+    /**
+     * Sets the {@code scheduleTime} of the {@code Person} that we are building.
+     */
+    public PersonBuilder withScheduleTime(String scheduleTime) {
+        this.schedule = new Schedule(this.schedule.scheduleName, this.schedule.dateString, scheduleTime);
+        return this;
     }
```

Comment:
Would it be better if these 3 methods are combined?

Reply:
I did consider that, but the other fields like `Phone`, `Email`, etc, only rely on a single string to create. However, for our `schedule`, there are three different string fields that can create different `schedule` objects. So I think having these three helper util methods would help us test each aspect individually later on